### ISSUE TITLE
GitHubSource: Creation of GitHubSource instances fail

### ISSUE THREAD

**Issue by Fabian-K** [knative/eventing-contrib#1084]
**Describe the bug**
I´ve applied a fix for #1048 locally such that the webhooks can be successfully registered. After this, the creation of new GitHubSources fail with 
````
failed calling webhook "defaulting.webhook.github.sources.knative.dev": Post https://github-webhook.knative-eventing.svc:443/defaulting?timeout=30s: service "github-webhook" not found
````
Looks like a service definition is missing that so far was not noticed as the webhooks were not registered due tu #1048 ?

 **To Reproduce**
- install github source
- add rbac permissions for webhook registration
- try to create a new GitHubSource

 **Knative release version**
v0.13.3

Thanks,
Fabian

**Comment by tom24d** [comment]
I think this is fixed by #1059 .

**Comment by pierDipi** [comment]
Fixed in #1059 
/close
